packages/axios-extension/src/abap/adt-catalog/handlers/transport.ts

```diff
@@ -11,7 +12,7 @@ import type { Logger } from '@sap-ux/logger';
  * @param log Service provider logger
  * @returns a list of valid transport numbers can be used for deploy config
  */
-export function getTransportNumberList(xml: string, log: Logger): string[] {
+export function getTransportNumberList(xml: string, log: Logger): TransportRequest[] {
```

Comment:
This function returns more than the transport number list now. Does `getTransports()` or `getTransportList()` work better?

Reply:
> > #518
> > Change API of ADT request getTransportRequests to return transport req metadata associated with transport numbers
> 
> @slin-sap, do you mind adding a brief reason for this change? The issues and PRs are good documentation to trace the changes to the code. Thanks


See updated the PR description